.env.development.example

```
@@ -0,0 +1,18 @@
+# Elzatona Web - Development Environment Template (.env.development)
+# Copy this to .env.development and fill in your development project credentials
+
+# Supabase Configuration (Development Project)
+NEXT_PUBLIC_SUPABASE_URL=https://your-dev-project.supabase.co
+NEXT_PUBLIC_SUPABASE_ANON_KEY=your-dev-anon-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_ANON_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)